elements/urth-core-watch/urth-core-watch.html

```diff
@@ -0,0 +1,43 @@
+<!--
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+-->
+<link rel="import" href="../polymer/polymer.html">
+<!--
+Creates an element that listens to changes to the `value` property. When it changes, it fires 'watch_notify' custom
+event on child elements.
+
+The element provides a declarative way of handling changes to {{variables}}.
```

Comment:
Would be good to include example usage in this comment like other elements do since it will get displayed in the built documentation.